### PR TITLE
Use connect to create containers.

### PR DESCRIPTION
**Pull request by paigeegorry**

### CODE REVIEW

File: src/components/markdown/Markdowns.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import Markdown from './Markdown';
 function Markdowns({
   markdowns
 }) {
+  console.log('list of markdowns', markdowns);
```
Comment (MariahAdams) [inline src/components/markdown/Markdowns.js]: Clean up those console logs!

File: src/components/markdown/Markdowns.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import Markdown from './Markdown';
 function Markdowns({
   markdowns
```
Comment (MariahAdams) [inline src/components/markdown/Markdowns.js]: Do you need to pass the whole markdown object, or just the title?

Comment (paigeegorry) [reply src/components/markdown/Markdowns.js]: We have to map through the array of markdowns, so we need the entire array.

File: src/components/markdown/Markdowns.js (hunk 1 above)
Comment (paigeegorry) [reply src/components/markdown/Markdowns.js]: Cleaned up!

**Pull request merged**